plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -0,0 +1,30 @@
+# @package _group_
+
+# This yaml will be validated against RayAWSLauncherConf
+
+ray_cluster_cfg:
+  provider:
+    key_pair:
+      key_name: "hydra"
+
+  # List of shell commands to run when calling ray up.
+  setup_commands:
+    # Please make sure the cluster python version matches your local machine version
+    - conda create -n hydra_${python_version:micro} python=${python_version:micro} -y
+    - echo 'export PATH="$HOME/anaconda3/envs/hydra_${python_version:micro}/bin:$PATH"' >> ~/.bashrc
+    # install needed dependency for launcher.
+    - python3 -m pip install --ignore-installed PyYAML
+    - python3 -m pip install ray==1.0.0
+    - python3 -m pip install cloudpickle==1.4.1
+    - python3 -m pip install git+https://github.com/facebookresearch/hydra.git@master
+    - python3 -m pip install -U https://hydra-test-us-west-2.s3-us-west-2.amazonaws.com/hydra_ray_launcher-0.1.0-py3-none-any.whl
+
```

Comment:
1. installing hydra would bring omegaconf which would bring pyyaml.
2. installing Hydra from master is problematic. we should probably install the same version the client is running (be it a specific pip package or a revision from GitHub).
3. Who is building the wheel for the plugin? what guarantees that it's the same version as the one on the client?

Reply:
> 1. installing hydra would bring omegaconf which would bring pyyaml.

fixed. I had some issue with the base AMI before, this is no longer needed.

> 2. installing Hydra from master is problematic. we should probably install the same version the client is running (be it a specific pip package or a revision from GitHub).

agreed. I want to add a custom resolver on getting current software version so will just work for all the software versions we care about. something like this #777, thoughts?

> 3. Who is building the wheel for the plugin? what guarantees that it's the same version as the one on the client?

I'm building the wheels for now manually. But i'm planning on automating that soon.